### ISSUE TITLE
Error storing file names containing unicode characters in database

### ISSUE THREAD

**Issue by deralexxx** [viper-framework/viper#106]
Tried to iplort a file with japanese / chinese chars as filename, the file is somewhat imported, but not storred completely, re-import is not possible. 

```
    _                   
    (_) 
```

---

  | | | | |  _ | ___ |/ **_)
   \ V /| | |_| | __**| |  
    _/ |_|  __/|_____)_| v1.1-dev
          |_|

You have 0 files in your default repository
viper > store -f ./to_upload/
[+] Stored file "何應对国.exe" to /home/xxx/viper2/viper/binaries/2/1/c/9/21c9a43d22cd02babb34b45a9defb881ae8228f0d034a0779b1321e851cad6a4
viper > find latest
+---+------+------+-----+------+------------+
| # | Name | Mime | MD5 | Tags | Created At |
+---+------+------+-----+------+------------+
+---+------+------+-----+------+------------+
viper > exit
xxx@zoo:~/viper2/viper$ ./viper.py 
         _  
        (_) 

---

  | | | | |  _ | ___ |/ **_)
   \ V /| | |_| | __**| |  
    _/ |_|  __/|_____)_| v1.1-dev
          |_|

You have 0 files in your default repository
viper > 
xxx@zoo:~/viper2/viper$ md5sum to_upload/*
76931daaa7c7f134531e7938b3a9052c  to_upload/何應对国.exe
xxx@zoo:~/viper2/viper$ ls -hall to_upload/
total 12K
drwxr-xr-x 2 xxx xxx 4.0K Aug 11 16:25 .
drwxr-xr-x 9 xxx xxx 4.0K Aug 11 16:25 ..
-rw-r--r-- 1 xxx xxx    6 Aug 11 16:25 何應对国.exe

**Comment by kevthehermit** [comment]
Whilst not ideal for retaining the original name you could use the string_clean function. This would at least enable it to be stored. then warn that a file was renamed

**Comment by cvandeplas** [comment]
I have added a pull-request that should fix this issue.


Some more info given in the pull-request:
I had a look at this as this is affecting me too. 
The main issue is that the `__sessions__.current.file.name` is used in different ways around in the code. (str, unicode). I would use unicode everywhere and only convert when legacy functions (like raw_imput) can't handle it. Unfortunately it is a massive and unrealistic job to fix this everywhere. 

To resolve this I have added the verification and conversion steps in the most efficient locations.
Going from std_in, the storing of files and the database function.

I have tested opening files (with and without special chars), storing them, loading them from the db, and it seems to work. As a test I have also ran multiple modules with no issue.

I expect some issues to occur here and there, but only with files containing special chars. So this patch should not break existing functionality, but instead extend it.
Mostly solution for the encountered issues will be to encode the unicode variable as can be seen below.
```python
__sessions__.current.file.name.encode('utf-8')
```
